### ISSUE TITLE
New SDK for the Ronin R 

### ISSUE THREAD

**Issue by alxbouchard** [Itamare4/dji_ronin#1]
There's a new SDK for the Ronin R series with communication and control over CAN BUS:
https://terra-1-g.djicdn.com/851d20f7b9f64838a34cd02351370894/DJI%20R%20SDK/DJI_R_SDK.zip
<img width="665" alt="Screen Shot 2020-11-20 at 2 04 39 PM" src="https://user-images.githubusercontent.com/21181350/99887440-77732380-2c12-11eb-8b7d-691b2be5d207.png">

**Comment by Itamare4** [comment]
Hi @alxbouchard, thanks for the info!

This package was built to fit the Ronin series M, 4 years ago it was the only way to control the system without  'hacking'/replacing the controller.

Thank goodness the DJI team finally decided to release an official SDK to communicate with the controller.

**Comment by ceinem** [comment]
Hi @alxbouchard 
Did you by any chance ever work with that new R SDK for the RS2? I've been trying for a couple of days now without success to get it working. My goal would be to turn it into a ROS package as soon as I have some basic functionality working so others can benefit as well.

**Comment by caseybasichis** [comment]
> Hi @alxbouchard 
> 
> Did you by any chance ever work with that new R SDK for the RS2? I've been trying for a couple of days now without success to get it working. My goal would be to turn it into a ROS package as soon as I have some basic functionality working so others can benefit as well. 

Did you ever make any progress with this?

**Comment by ceinem** [comment]
Yes, I got it working and managed to implement some functions. It can be found [here.](https://github.com/ceinem/dji_rs2_ros_controller)
